server/backend/check.ml

```diff
@@ -168,7 +167,24 @@ let failure_kind conf ~pkg logfile =
       | None -> Lwt.return res
     in
     lookup `Other
-  end
+
+let failure_kind_dune logfile =
+  Lwt_io.with_file ~mode:Lwt_io.Input (Fpath.to_string logfile) @@ fun ic ->
+    let rec lookup res =
+      let* line = Lwt_io.read_line_opt ic in
+      match line with
+      | Some "opam-health-check: Build failed" -> Lwt.return `Failure
```

Comment:
Won't this hide the reason of the failure?

Reply:
This is just what a build failure gets classified at (so "white", "grey", "red" in the legend), the logfile still stays available.

Reply:
OK, just wanted to make sure!